recipe/meta.yaml

```diff
@@ -13,11 +13,10 @@ source:
   sha256: a9e3acc58d8d67a08dbe6c86513b6bbff734a608a7fceb102883a88877643059
 
 build:
-  skip: true  # [python_impl == 'pypy']
   number: 1
   script:
-    - copy /y "{{ LIBRARY_INC }}\capnp\*.capnp" .\capnp\  # [win]
-    - cp -f {{ PREFIX }}/include/capnp/*.capnp ./capnp/  # [not win]
+    - copy /y "{{ environ.get('LIBRARY_INC', '') }}\capnp\*.capnp" .\capnp\  # [win]
+    - cp -f "{{ environ.get('PREFIX', '') }}/include/capnp"/*.capnp ./capnp/  # [not win]
```

Comment:
Wrapped these Jinja variables with `environ.get(...)`. That way they are allowed to be undefined during re-rendering, migrations, etc..

At build time, these use the actual variable values